services/app-api/forms/sar.json

```diff
@@ -1396,64 +1396,45 @@
-                      }
-                    ]
-                  },
-                  {
-                    "type": "li",
-                    "content": "State/Territory equivalent funds attributable to the MFP-enhanced match"
-                  }
-                ]
-              },
-              {
-                "type": "p",
-                "content": "State or territory-specific initiatives are a distinct set of activities designed to increase the use of HCBS rather than institutional long-term services and supports (LTSS). These initiatives are specified in your MFP Work Plan and imported into the form below."
-              },
-              {
-                "type": "p",
-                "content": "Recipients must report on the progress of initiatives that were ongoing during the current reporting period. For each initiative, enter information on expenditures and activities, whether continuing from prior reporting periods or initiated during this reporting period."
-              },
-              {
-                "type": "p",
-                "content": "For each initiative, recipients must also report on progress toward achieving the objective(s) identified in the MFP Work Plan. Progress towards these objectives indicates the state or territory's greater ability to provide HCBS instead of services in institutional settings. If your state or territory has not achieved the targets for performance measures or expected time frames for deliverables set in the MFP Work plan, use the following questions to explain the barriers or challenges that have hindered progress and describe plans to address them."
-              }
-            ]
-          }
+          "subsection": "State- or Territory-Specific Initiatives"
+        },
+        "accordion": {
+          "buttonLabel": "Instructions",
+          "intro": [
+            {
+              "type": "html",
+              "content": "This section requests information on current, new, or expanded initiatives implemented under the MFP demonstration. These initiatives can be funded using one or more of these funding sources:"
+            }
+          ],
+          "list": [
+            "MFP cooperative agreement funds for:",
+            [
+              "Qualified HCBS and demonstration services",
+              "Supplemental services",
+              "Administrative activities",
+              "Capacity building initiatives"
+            ],
+            "State/Territory equivalent funds attributable to the MFP-enhanced match"
+          ],
+          "text": [
+            {
+              "type": "html",
+              "content": "State or territory-specific initiatives are a distinct set of activities designed to increase the use of HCBS rather than institutional long-term services and supports (LTSS). These initiatives are specified in your MFP Work Plan and imported into the form below."
+            },
+            {
+              "type": "html",
```

Comment:
I think you can swap all of the html types for div's here (lines 1405, 1421, 1425, and 1429). Theres not really any html code we want to render, and the div can handle the <br> in there as well. Plus if you use a div, you need 1 less br in the content!